compose.dev.yml

```diff
@@ -65,7 +65,7 @@ services:
       context: .
       dockerfile: Containerfile
     container_name: recordtransfer_django_app
-    command: python manage.py runserver 0.0.0.0:8000
+    command: /bin/sh -c "npm run build && python manage.py runserver 0.0.0.0:8000"
```

Comment:
Just taking a quick look at your changes so far, `npm run build` should be run from `entrypoint.sh` instead of here.

Reply:
Or actually, I think I see why this is here. If you're developing the JavaScript you'd need to be able to re-run the webpack command

Reply:
I think it should be fine if I move this to `entrypoint.sh` - I tried it and there were no issues. I'm not sure what you mean by being able to run the webpack command.

Reply:
The webpack command I'm referring to is what's being run during `npm run build` - i.e., `webpack --config webpack.config.js`

Reply:
Ah I see what you mean. Moving `npm run build` to the entrypoint would not be an issue since npm scripts can be run from anywhere within a node project.